.github/workflows/ci.yml

```diff
@@ -8,7 +8,21 @@ jobs:
   Test_pypackage:
     name: Test pypackage
     runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:15.3-alpine
+        ports:
+        - 5432:5432
+        env:
+          POSTGRES_HOST_AUTH_METHOD: trust
+        options:
+          --health-cmd pg_isready
+          --health-interval 1s
```

Comment:
The cookiecutter's own `ci.yml` needs Postgres now: what the cookiecutter's tests do is use the cookiecutter to generate new projects and then run the tests of those new projects, and the tests of freshly-cookiecuttered projects now try to connect to the DB.

Note the healthcheck stuff: I ran into a race condition where it would get to the create DB commands below before Postgres had finished starting up. This makes it wait. We might want to consider adding this to our other projects because they have the same race condition, we've just never noticed because they do other things (checking out the git repo etc) before trying to create the test DBs.